packages/dds/tree/src/events/events.ts

```diff
@@ -59,7 +59,10 @@ export type TransformEvents<E extends Events<E>, Target extends IEvent = IEvent>
     : never;
 
 /**
- * An object which allows the registration of listeners so that subscribers can be notified when an event happens
+ * An object which allows the registration of listeners so that subscribers can be notified when an event happens.
```

Comment:
The docs here are somewhat redundant with the ones for the "on" function.

I think it would be better to just have the docs about "on" on the "on" method (but fix those docs so the end with a period after the sentences).